cli/src/test/java/org/jboss/pnc/bacon/cli/AppTest.java

```diff
@@ -1,47 +1,56 @@
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 class AppTest {
 
     private static final String PNC_TEST_CLASSES = FilenameUtils.separatorsToSystem("pnc/target/test-classes");
 
     @Test
     void testHelp() throws Exception {
-        App app = new App();
-        app.run(new String[] { "-h" });
-        String text = tapSystemOut(() -> assertEquals(0, app.run(new String[] { "-h" })));
-        assertTrue(text.contains("Usage: bacon [-hovV] [-p=<configurationFileLocation>] [--profile=<profile>]"));
+        restoreSystemProperties(() -> {
+            System.setProperty("picocli.ansi", "false");
+            App app = new App();
+            app.run(new String[] { "-h" });
+            String text = tapSystemOut(() -> assertThat(app.run(new String[] { "-h" })).isZero());
+            assertThat(text).contains("Usage: bacon [-hovV] [-p=<configurationFileLocation>] [--profile=<profile>]");
+        });
     }
 
     @Test
     void testExampleUsage() throws Exception {
         App app = new App();
         String text = tapSystemOut(
-                () -> assertEquals(
-                        0,
-                        app.run(new String[] { "--verbose", "pnc", "admin", "maintenance-mode", "activate", "-h" })));
-        assertTrue(text.contains("bacon pnc admin maintenance-mode activate \"Switching"));
+                () -> assertThat(
+                        app.run(new String[] { "--verbose", "pnc", "admin", "maintenance-mode", "activate", "-h" }))
+                                .isZero());
+        assertThat(text).contains("bacon pnc admin maintenance-mode activate \"Switching");
     }
 
     @Test
     void testConfigPath() throws Exception {
         String text = tapSystemErr(
-                () -> assertEquals(
-                        0,
-                        new App().run(new String[] { "-p", "/tmp/123456789", "-v", "pnc", "-o", "build", "-h" })));
-        assertTrue(text.contains("Config file set from flag with profile default to /tmp/123456789/config.yaml"));
+                () -> assertThat(
+                        new App().run(
+                                new String[] {
+                                        "-p",
+                                        FilenameUtils.separatorsToSystem("/tmp/123456789"),
+                                        "-v",
+                                        "pnc",
+                                        "-o",
+                                        "build",
+                                        "-h" })).isZero());
+        assertThat(text).contains(
+                "Config file set from flag with profile default to "
+                        + FilenameUtils.separatorsToSystem("/tmp/123456789/config.yaml"));
```

Comment:
All of the other paths were (correctly) converted using `FilenameUtils.separatorsToSystem()`. However, this one was missed. Hardcoding the path here is fragile in general and should probably use `@TempDir` https://junit.org/junit5/docs/5.4.0/api/org/junit/jupiter/api/io/TempDir.html.

Reply:
I was just examining the log for that profile but ok

Reply:
I didn't actually check if `-p` creates the file or not. If it's not actually creating it, just logging it, then it's OK to set it to something "fake" like you have it. But, I can also change it to use `@TempDir` easily.